CHANGELOG.md

```diff
@@ -6,6 +6,42 @@
   useful summary for people upgrading their application, not a replication
   of the commit log.
 
+## 1.0.0
+
+This release upgrades to Stylelint 15 and now includes:
+
+* Checks for media query syntax, rules and values
+* Checks for number precision inside CSS functions
+* Fixes for duplicate rules with intentionally different units
+
+Read about breaking changes in the Stylelint [**Migrating to v15.0.0**](https://github.com/stylelint/stylelint/blob/main/docs/migration-guide/to-15.md#breaking-changes) guide.
+
+### Support for Internet Explorer 8, 9 and 10
+
+Support for IE8–10 has not been a requirement for services since June 2018, so we removed the rule [`selector-pseudo-element-colon-notation`](https://stylelint.io/user-guide/rules/selector-pseudo-element-colon-notation/) preventing double-colon `::before` and `::after` pseudo element notation.
+
+This change was made in [pull request #36: Remove `selector-pseudo-element-colon-notation`](https://github.com/alphagov/stylelint-config-gds/pull/36).
```

Comment:
```suggestion
### Removal of rule enforcing single colon for pseudo selectors

We removed the rule [`selector-pseudo-element-colon-notation`](https://stylelint.io/user-guide/rules/selector-pseudo-element-colon-notation/) preventing double-colon `::before` and `::after` pseudo element notation as this was needed for IE8-10 support and supporting these browsers has not been a requirement for services since June 2018.

This change was made in [pull request #36: Remove `selector-pseudo-element-colon-notation`](https://github.com/alphagov/stylelint-config-gds/pull/36). If you wish to keep support for these old browsers you can [configure your Stylelint](https://stylelint.io/user-guide/configure/) to re-enable this rule.
```

I think by saying support for IE8-10 in the title, the change sounds a lot more drastic than it is, so think just flagging the single rule is a bit clearer.

Reply:
Mind taking a look at https://github.com/alphagov/stylelint-config-gds/pull/44/commits/c5d793e22aff0568f20b821ca401f9104f225993 over in #44? Thanks @kevindew

Gone for something very similar but included the single-colon `:before` and `:after` examples now